addons/binding/org.openhab.binding.windcentrale/examples.txt

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
A nicer and more functional way to add such information would be if it was part of a unit test.